ddtrace/tracer.py

```diff
@@ -142,8 +155,8 @@ def context_provider(self):
 
     # TODO: deprecate this method and make sure users create a new tracer if they need different parameters
     def configure(self, enabled=None, hostname=None, port=None, uds_path=None, https=None,
-                  dogstatsd_host=None, dogstatsd_port=None, sampler=None, context_provider=None,
```

Comment:
we can't remove `dogstatsd_host` and `dogstatsd_port` from here, breaks the API.

Reply:
it is also tricky, because if they did supply `hostname='value'` I wonder if we should inherit that for dogstatsd as well.

Reply:
👍